samples/contrib/aws-samples/mnist-kmeans-sagemaker/README.md

```diff
@@ -45,7 +45,7 @@ You will also have an model endpoint in service. Refer to [Prediction section](#
   - Opening SageMaker [console](https://us-east-1.console.aws.amazon.com/sagemaker/home?region=us-east-1#/endpoints),  or
   - Clicking the `sagemaker-deploy-model-endpoint_name` under `Output artifacts` of `SageMaker - Deploy Model` component of the pipeline run
 
-2. Setup AWS credentials with `sagemaker:InvokeEndpoint` access. [Sample commands](https://sagemaker.readthedocs.io/en/stable/workflows/kubernetes/using_amazon_sagemaker_components.html#configure-permissions-to-run-predictions)
+2. Setup AWS credentials with `sagemaker:InvokeEndpoint` access. [Sample commands](https://docs.aws.amazon.com/sagemaker/latest/dg/kubernetes-sagemaker-components-tutorials.html)
```

Comment:
Refer to the `Configure permissions to run predictions` section in [AWS docs](https://docs.aws.amazon.com/sagemaker/latest/dg/kubernetes-sagemaker-components-tutorials.html) if you need sample commands